app/src/main/java/com/keylesspalace/tusky/fragment/AccountMediaFragment.kt

```diff
@@ -235,14 +234,6 @@ class AccountMediaFragment : BaseFragment(), RefreshableFragment, Injectable {
             topProgressBar?.show()
     }
 
-    // That's sort of an optimization to only load media once user has opened the tab
-    // Attention: can be called before *any* lifecycle method!
-    override fun setUserVisibleHint(isVisibleToUser: Boolean) {
-        super.setUserVisibleHint(isVisibleToUser)
-        this.isVisibleToUser = isVisibleToUser
-        if (isVisibleToUser && isAdded) doInitialLoadingIfNeeded()
-    }
```

Comment:
This does not work anymore with ViewPager2. I don't think its a problem as ViewPager2 seems to only start the fragment once its visible.